torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -367,15 +367,21 @@ def affine_bounding_box(
 
 
 def affine_segmentation_mask(
-    mask: torch.Tensor,
+    segmentation_mask: torch.Tensor,
     angle: float,
     translate: List[float],
     scale: float,
     shear: List[float],
     center: Optional[List[float]] = None,
 ) -> torch.Tensor:
-    return affine_image_tensor(
-        mask,
+    if segmentation_mask.ndim < 3:
```

Comment:
I would love to do that in a decorator, but `torch.jit.script` doesn't support it :cry: 

```py
def handle_segmentation_mask(mask_kernel):
    @functools.wraps(mask_kernel)
    def wrapper(mask, *other_args, **kwargs):
        if mask.ndim < 3:
            mask = mask.unsqueeze(0)
            needs_squeeze = True
        else:
            needs_squeeze = False

        output = mask_kernel(mask, *other_args, **kwargs)

        if needs_squeeze:
            output = output.squeeze(0)

        return output

    return wrapper


@handle_segmentation_mask
def affine_segmentation_mask(mask, ...):
    ...
```

Reply:
The story of our lives...